src/main/java/uk/gov/hmcts/reform/translate/security/SecurityUtils.java

```diff
@@ -76,5 +77,9 @@ public String getUserRolesHeader() {
             .map(GrantedAuthority::getAuthority)
             .collect(Collectors.joining(","));
     }
+
+    public boolean hasManageTranslationsRole(List<String> roles, String role) {
```

Comment:
This has been updated on `WLTS-5`, which has now been merged to `master`

Reply:
I merged from master and I am using this securityUtils.hasRole() .

Reply:
DONE